From: =?utf-8?q?Louis-Philippe=5FV=C3=A9ronneau?= <pollo@debian.org>
Date: Tue, 12 Mar 2019 13:51:37 -0400
Subject: Use python3 by default instead of /usr/bin/env python

Forwarded: not-needed
Last-Update: 2019-06-08

On systems running both python2 and python3 and where python2 is the default,
running 'example_project/manage.py' fails, since this is a python3 package.
---
 example_project/manage.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/example_project/manage.py b/example_project/manage.py
index b5a2943..e9b9ab0 100755
--- a/example_project/manage.py
+++ b/example_project/manage.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 import os
 import sys
 
